jakarta.ls/pom.xml

```diff
@@ -58,7 +58,7 @@
     <dependency>
       <groupId>junit</groupId>
       <artifactId>junit</artifactId>
-      <version>4.11</version>
+      <version>4.13.2</version>
```

Comment:
I think this also needs to change here: [jakarta.eclipse/org.eclipse.lsp4jakarta.lsp4e.test/META-INF/MANIFEST.MF](https://github.com/mezarin/lsp4jakarta-test/blob/41b39ec679f2b76c5b50eded9628d739b24978d6/jakarta.eclipse/org.eclipse.lsp4jakarta.lsp4e.test/META-INF/MANIFEST.MF#L8)